modrequest.go

```diff
@@ -0,0 +1,14 @@
+package main
+
+import (
+    "fmt"
+
+    "github.com/bwmarrin/discordgo"
+)
+
+const modRequest = "<@%s> requested a moderator in <#%s> here: %s\nReason: %s"
+
+func (b *bot) requestMod(s *discordgo.Session, m *discordgo.Message, parts []string) {
```

Comment:
The change from will look like:

```suggestion
func (b *bot) requestMod(m *discordgo.Message, parts []string) {
```